### PR TITLE
Add NLP scripts

### PR DESCRIPTION
**Pull request by akzaidi**
Adds NLP scripts
I tested and it should work, but could you test again if you have time?
I added the `Rproj` file back in so we can easily pull in new changes if some have already setup
I didn't change the setup script, to avoid running additional script actions
Data is small so we will download in class
I need three packages: ggrepel, devtools (for d3wordcloud), and d3wordcloud. All should download within <2 minutes, but perhaps we can skip 'em for time.

### CODE REVIEW

**Comment by akzaidi** [comment]
I have no idea why the msfclas bot thinks I'm a vendor...

File: Code/NLP/0-ingest-data.R [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## download data
+download.file("https://alizaidi.blob.core.windows.net/training/data/imdb-train.xdf",
+              "imdb-train.xdf")
+
+download.file("https://alizaidi.blob.core.windows.net/training/data/imdb-test.xdf",
+              destfile = "imdb-test.xdf")
+
+## make local pointers
+train_xdf <- RxXdfData("imdb-train.xdf")
```
Comment (inchiosa) [inline Code/NLP/0-ingest-data.R]: should we set the working dir?

File: Code/NLP/0-ingest-data.R [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+test_xdf <- RxXdfData("imdb-test.xdf")
+
+## make hdfs pointers
+hdfs <- RxHdfsFileSystem()
+
+train_hdfs <- RxXdfData('/imdbdata/train/',
```
Comment (inchiosa) [inline Code/NLP/0-ingest-data.R]: Do we need a command to create the /imdbdata dir?

File: Code/NLP/0-ingest-data.R [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                        fileSystem = hdfs)
+test_hdfs <- RxXdfData('/imdbdata/test/',
+                       fileSystem = hdfs)
+
+## import data to hdfs
+rxDataStep(inData = train_xdf,
```
Comment (inchiosa) [inline Code/NLP/0-ingest-data.R]: May want to add overwrite = T

File: Code/NLP/1-SentimentAnalysis.Rmd [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    scale_size(range = c(2, 15), guide = FALSE) +
+    scale_y_continuous(breaks = NULL) +
+    scale_x_continuous(breaks = NULL) +
+    labs(x = '', y = '') +
+    theme_classic() +
+    facet_wrap(~sentiment)
+
+```
+
+Let's try and makea  more interactive visual. We'll use `purrr` again to map our coefficients to the beautiful [d3wordcloud](https://github.com/jbkunst/d3wordcloud) package
```
Comment (inchiosa) [inline Code/NLP/1-SentimentAnalysis.Rmd]: typo: makea -> make a

**Pull request merged**